README.md

```diff
@@ -0,0 +1,44 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio Design
+
+> This project is built to create the mobile-version design of my porfolio.
+
+
+## Built With
+
+- HTML & CSS
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
```

Comment:
[Optional] The previous reviewer requested, you add instructions on how to get started. You could add more information such as:

- Download the zip file
- Run HTML file
- Use inspect browser tools to view mobile version